src/plib/resources/locales/en-US.php

```diff
@@ -42,7 +43,8 @@
     'syncAllDone' => 'All zones registered in Plesk were synchronized with Amazon.',
     'removeAllDone' => 'All zones hosted on Amazon were removed.',
     'awsApiAuthorization' => 'AWS API Authorization',
-    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon. If you provide root account credentials, Plesk will create a restricted IAM user account for you.',
+    'awsDescriptionWindow' => 'To manage Route53, you need a restricted IAM user account to authenticate with Amazon.',
```

Comment:
* What does "restricted IAM user account" mean? I can't find such term for Amazon.
* Does it all mean that one needs to authenticate in AWS using such account before managing Route 53?

Reply:
Restricted user is a user with limited rights. 
Before fix there was an ability to create root account in AWS using extension and login with this user or use only 'limited' account. Now we allow to login only the second way of authentication.

https://docs.aws.amazon.com/signin/latest/userguide/console-sign-in-tutorials.html

Reply:
You need an IAM user to manage Amazon Route 53.